src/common/SurgeSynthesizer.cpp

```diff
@@ -1904,7 +1908,9 @@ void SurgeSynthesizer::getParameterMeta(long index, parametermeta& pm)
       pm.fdefault = 0.5f;
       pm.hide = false;
       pm.meta =
-          false; // ironiskt eftersom det �r metaparameters, men dom p�verkar inga andra sliders
+          false;// ironiskt eftersom det är metaparameters, men dom påverkar inga andra sliders
+	   	// TRANSLATE:
+	   	// ironic because it is metaparameters, but they don't affect any other sliders
```

Comment:
@esaruoho not sure if @kurasu would mind just translating and removing the non-English comments? probably best to allow for the most people to read/contribute, but not my call necessarily

Reply:
@kzantow well, i left some of the swedish in because i think @kurasu has the final say whether the translation is accurate or inaccurate. some of it has been ripped off, if it's absolutely certain that it's true. i would, ideally, remove all the swedish out, but would rather not first remove it and then have to work with inaccurate translations because turns out that something that said "this is NOT required" actually meant "this is ABSOLUTELY required" :)